### PR TITLE
Deposit Mandatory

### PR DESCRIPTION
**Pull request by Bagaric**
- Make deposit mandatory with a minimum required deposit
- Add missing packages to package.json

If merging please squash commits.

### CODE REVIEW

**Comment by Bagaric** [comment]
@kosecki123 @lsaether The current minimum deposit I set is 1 Gwei. Let me know if there is a problem with that and I can adjust the value.

File: cli/eac.js [hunk 1]
```diff
@@ -98,6 +98,16 @@ const getDefaultSchedulingValues = async () => {
   } 
 };
 
+const readUntilValid = (funcName) => {
+  while (true) {
```
Comment (adibas03) [inline cli/eac.js]: won't this lead to an eternal loop?

Comment (Bagaric) [reply cli/eac.js]: It will look until it meets the return condition, which happens after they input a correct value.

Comment (Bagaric) [reply cli/eac.js]: Might be a good idea to limit the loop to a couple of tries though.

**Comment by lsaether** [comment]
I don't see a reason to hard code a minimum deposit.

**Comment by Bagaric** [comment]
Orphaned. Closing.

**Pull request closed without merging**